org.eevolution.warehouse/src/main/java/base/org/spin/form/OutBoundOrder.java

```diff
@@ -166,19 +188,21 @@
 		ResultSet rs = null;
 		PreparedStatement pstmt = null;
 		if(docTypeTargetId > 0) { 
-			MDocType m_DocType = MDocType.get(Env.getCtx(), docTypeTargetId);
-			validateQuantity = m_DocType.get_ValueAsBoolean("IsValidateQuantity");
+			MDocType documentType = MDocType.get(Env.getCtx(), docTypeTargetId);
+			validateQuantity = documentType.get_ValueAsBoolean("IsValidateQuantity");
 		}
 		//	
 		Vector<Vector<Object>> data = new Vector<Vector<Object>>();
 		StringBuffer sql = null;
 		if (movementType.equals(I_DD_Order.Table_Name)) {
 			//Query for Material Movement
 			sql = new StringBuffer("SELECT " +
-					"wr.Name Warehouse, ord.DD_Order_ID, ord.DocumentNo, " +	//	1..3
-					"ord.DateOrdered, ord.DatePromised, reg.Name, cit.Name, sr.Name SalesRep, " +	//	4..8
-					"cp.Name Partner, bploc.Name, " +	//	9..10
-					"loc.Address1, loc.Address2, loc.Address3, loc.Address4, ord.C_BPartner_Location_ID, ord.Weight, ord.Volume " +	//	11..17
+					"ord.DD_Order_ID, ord.DocumentNo, ord.Description , cp.Name Partner, bploc.Name, wr.Name Warehouse, " +
```

Comment:
I don't sure if this is correct, maybe is a change for a customer or own change, the Weigh and Volume is showed from Order to select instead of Quantoty to deliver.

I think that this change can be part of a ADempiere Labs, Customizxation for your customer otr a Location